src/app/split-view/multi/terra-multi-split-view.component.scss

```diff
@@ -4,7 +4,8 @@
     //overflow: hidden;
     padding: 0;
     margin: 0;
-    height: 33px;
+    //height: 33px;
```

Comment:
der auskommentierte Wert ist wesentlich und die Änderung hat Auswirkungen auf die Positionierung des Kontext-Menüs der Data-Table, deshalb passe ich noch eine weitere Datei an und push den commit in deinen Branch